AlfrescoSDK/AlfrescoSDK/Constants/AlfrescoConstants.h

```diff
@@ -314,3 +314,20 @@ extern NSString * const kAlfrescoWorkflowVariableTaskInvitePendingOutcome;
 extern NSString * const kAlfrescoWorkflowTaskTransitionApprove;
 extern NSString * const kAlfrescoWorkflowTaskTransitionAccept;
 extern NSString * const kAlfrescoWorkflowTaskTransitionReject;
+
+/**---------------------------------------------------------------------------------------
+ * @name Connection Diagnostic Constants
+ --------------------------------------------------------------------------------------- */
+extern NSString * const kAlfrescoConfigurationDiagnosticDidStartEventNotification;
+extern NSString * const kAlfrescoConfigurationDiagnosticDidEndEventNotification;
+
+extern NSString * const kAlfrescoConfigurationDiagnosticDictionaryIsLoading;
+extern NSString * const kAlfrescoConfigurationDiagnosticDictionaryIsSuccess;
```

Comment:
Instead of having separate keys for isLoading and isSuccess, should we have a "status" key where the value would then be set to either "Loading" or "Successful"? This way the dictionary will be easier to manager if we require more states?